packages/webdriverio/tests/node/utils.test.ts

```diff
@@ -5,24 +5,17 @@ import { getAbsoluteFilepath, assertDirectoryExists } from '../../src/node/utils
 
 describe('node utils', () => {
     describe('getAbsoluteFilepath', () => {
-        it('should not change filepath if starts with forward slash', () => {
-            const filepath = '/packages/bar.png'
-            expect(getAbsoluteFilepath(filepath)).toEqual(filepath)
-        })
-
-        it('should not change filepath if starts with backslash slash', () => {
-            const filepath = '\\packages\\bar.png'
-            expect(getAbsoluteFilepath(filepath)).toEqual(filepath)
-        })
-
-        it('should not change filepath if starts with windows drive letter', async () => {
-            const filepath = 'E:\\foo\\bar.png'
-            expect(getAbsoluteFilepath(filepath)).toEqual(filepath)
-        })
-
-        it('should change filepath if does not start with forward or back slash', async () => {
-            const filepath = 'packages/bar.png'
-            expect(getAbsoluteFilepath(filepath)).toEqual(path.join(process.cwd(), 'packages/bar.png'))
+        it.for([
+            '/packages/bar.png',
+            '\\packages\\bar.png',
+            'packages/bar.png',
+            'packages\\bar.png',
+            '\\\\packages\\bar.png',
+            '//packages/bar.png',
+            'E:/foo/bar.png',
+            'E:\\foo\\bar.png',
```

Comment:
These should include all the existing cases with the addition of the one with the drive and forward slashes, and the ones with a double back/forward slash at the start.

The test itself just checks that it works the same way as `path.resolve` does, hopefully to prevent introducing bugs in the future.